### PR TITLE
Handle empty KButtonGroups

### PR DESCRIPTION
**Pull request by rtibbles**
## Description
Catches an edge case when a KButtonGroup has no Buttons inside it.

## Steps to test

1. Create a KButtonGroup with nothing in it
2. Confirm it can render without errors.
3. Profit!

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change has been added to the `changelog`

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## Post-merge updates and tracking

- [ ] Learning Platform update PR is submitted
- [ ] Learning Platform update PR is merged
- [ ] Studio update PR is submitted
- [ ] Studio update PR is merged
- [ ] Data Portal update PR is submitted
- [ ] Data Portal update PR is merged

### CODE REVIEW

File: lib/buttons-and-links/KButtonGroup.vue [hunk 1]
```diff
@@ -11,7 +11,8 @@
     name: 'KButtonGroup',
     render(createElement) {
       var children = [];
-      this.$slots.default.forEach(element => {
+      // Add an existence catch in case nothing is passed into the slots.
```
Comment (nucleogenesis) [inline lib/buttons-and-links/KButtonGroup.vue:14]: lgtm!

**Review by nucleogenesis — APPROVED** [review]
